td.vue/src/background.js

```diff
@@ -0,0 +1,83 @@
+"use strict";
```

Comment:
can we rename this away from the default `background.js` ?
something with `electron` and `td` in it would indicate what the file is used for ... not worried what the name is so long as it is clear

Reply:
Agreed! How about renaming this to `td.desktop.js`?

Reply:
Yes, sure - `td.desktop.js` sounds good to me

Reply:
Done! Thank you for this suggestion too!